packages/react-core/src/components/MultipleFileUpload/__tests__/MultipleFileUploadStatusItem.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('MultipleFileUploadStatusItem', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
-  test('renders custom file name/size/icon when passed', () => {
+  test('renders custom file name/size/icon/ariaLiveMessage when passed', () => {
```

Comment:
Can we also add a test for when a function is passed to `progressAriaLiveMessage` and also for when `progressAriaLiveMessage`  is undefined?